src/features/Authentication/organisms/SignUpForm/SignUpForm.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { Formik, Form, Field } from 'formik';
+import * as Yup from "yup";
+import { TextField } from 'formik-mui';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import CircularProgress from '@mui/material/CircularProgress';
+import { CREATE_USER } from 'mutations/CreateUser/createUser';
+import { useMutation } from 'urql';
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+
+const SIGNUP_SCHEMA = Yup.object().shape({
+  firstName: Yup.string()
+    .min(2, 'Too Short!')
+    .max(70, 'Too Long!')
```

Comment:
2 and 70 can be extracted to the constants

Reply:
Error messages are not accessible. You need to specify required length in the error message.